### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by josteink**
Add some basic installation-instructions for new users.

Verified working for me on Ubuntu 18.04 LTS.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,3 +2,64 @@
...
+
+# obtain sample config, customize
+wget -O config.yaml https://raw.githubusercontent.com/Sorunome/mx-puppet-bridge/master/sample.config.yaml
+nano config.yaml
+
+# get dependencies
+npm install
+
+# compile typescript-code to javascript
+./node_modules/.bin/tsc
```
Comment (tulir) [inline README.md:32]:
```suggestion
npm run build
```

File: README.md [hunk 2]
```diff
@@ -2,3 +2,64 @@
...
+
+# generate registration file, deploy. adapt as needed
+node ./build/index.js -r
```
Comment (tulir) [inline README.md:35]:
```suggestion
# generate registration file, deploy. adapt as needed
npm start -r
```

File: README.md [hunk 3]
```diff
@@ -2,3 +2,64 @@
...
+sudo chown matrix-synapse /etc/matrix-synapse/instagram-registration.yaml 
+
+# manually add registration to synapse daemon. adapat as needed
+sudo nano /etc/matrix-synapse/homeserver.yaml 
+
+# restart matrix daemon
+sudo systemctl restart matrix-synapse
+
+# run actual bridge
+node ./build/index.js
```
Comment (tulir) [inline README.md:46]:
```suggestion
npm start
```

**Comment by josteink** [comment]
All reasonable comments. Will fix.

**Comment by josteink** [comment]
I've fixed a few typos I've found in the comments, and consider this ready for review/merging.

**Comment by josteink** [comment]
Any news on this one? Anything else I need to add?

**Comment by Sorunome** [comment]
Totally forgot this PR - sorry about that and thank you for the ping!

**Pull request merged**